README.md

```diff
@@ -11,30 +11,24 @@ The Snorkel tutorials are grouped by application, with some applications having
 
 See the [Tutorials Index](#tutorials-index) for a listing of which tutorials demonstrate which task types, techniques, and integrations.
 
-## Getting Started
-Start with the `spam` tutorial for a gentle introduction to the concepts and classes of Snorkel.
+We recommend that all users **start with the `spam` tutorial** for a gentle introduction to the concepts and classes of Snorkel.  
 All other tutorials assume that you have already completed that tutorial and are familiar with its concepts.
 
-To run a tutorial in a jupyter notebook, run the following commands, which create a virtual environment, install requirements, create an ipython kernel, and launch Jupyter:
-```bash
-TUTORIAL="spam"  # Change this to the name of the directory for the tutorial you want
+## Getting Started
+To run a tutorial, clone this repository and run the following commands (replacing `spam` with the name of the tutorial you want to run, if applicable). We recommend doing this in a virtual environment with e.g., `virtualenv` or `conda`.
```

Comment:
Add a line that says: For example if you use `pip`, first activate your virtualenv if you're using one, then run:

Reply:
Also add links to virtualenv docs and conda envs docs?

Reply:
Hm ok rereading, I'd say:

```
Running a tutorial has four required steps:

1. Cloning this repo
1. Installing repo-wide requirements
1. Installing tutorial-specific requirements
1. Launching a Jupyter notebook server or executing as a script

We recommend installing requirements in a virtual environment using [`virtualenv`](link) or [`conda`](link).
For example if you use pip, first activate your virtualenv if you're using one, then run:
```